rust/src/nfs/nfs.rs

```diff
@@ -1069,7 +1069,10 @@ impl NFSState {
                 SCLogDebug!("NFSv2 reply record");
                 return self.process_reply_record_v2(r, &xidmap);
             },
-            _ => { panic!("unsupported NFS version"); },
+            _ => {
+                SCLogNotice!("unsupported NFS version");
```

Comment:
Looks like having a NFSv4 message would be legit so we should ignore this value.

For weird value, using a log notice message could introduce a message storm if ever a lot of weird packets are sent. It is better to use a Debug level message and set an event instead with 
```
self.set_event(NFSEvent::MalformedData);
```
Ideally, a dedicated event like NonExistingVersion would be better.

Reply:
Changes seem simple enough, I'll get them sorted :)